packages/contracts/deploy-config-example.json

```diff
@@ -708,6 +708,208 @@
       "stateTreeDepth": 10
     }
   },
+  "polygon_cardona_zkevm": {
+    "ConstantInitialVoiceCreditProxy": {
+      "deploy": true,
+      "amount": 99
+    },
+    "ERC20VotesInitialVoiceCreditProxy": {
+      "deploy": false,
+      "token": "0x0000000000000000000000000000000000000000",
+      "snapshotBlock": 1,
+      "factor": 1
+    },
+    "ERC20Policy": {
+      "deploy": false,
+      "token": "0x0000000000000000000000000000000000000000",
+      "threshold": 1
+    },
+    "FreeForAllPolicy": {
+      "deploy": true
+    },
+    "MerkleProofPolicy": {
+      "deploy": false,
+      "root": "0x2461fcc4c0965cb7f482dd28f1ca8057b7a62a35e8b7a86bb3ad6523f4bb21c0"
+    },
+    "ERC20VotesPolicy": {
+      "deploy": false,
+      "token": "0x0000000000000000000000000000000000000000",
+      "threshold": 1,
+      "snapshotBlock": 1
+    },
+    "EASPolicy": {
+      "deploy": false,
+      "easAddress": "0x0000000000000000000000000000000000000000",
+      "schema": "0xe2636f31239f7948afdd9a9c477048b7fc2a089c347af60e3aa1251e5bf63e5c",
+      "attester": "0x0000000000000000000000000000000000000000"
+    },
+    "GitcoinPassportPolicy": {
+      "deploy": true,
+      "decoderAddress": "0x2443D22Db6d25D141A1138D80724e3Eee54FD4C2",
+      "passingScore": 5
+    },
+    "ZupassPolicy": {
+      "deploy": false,
+      "signer1": "13908133709081944902758389525983124100292637002438232157513257158004852609027",
+      "signer2": "7654374482676219729919246464135900991450848628968334062174564799457623790084",
+      "eventId": "d2ce5bb2-99a3-5a61-b7e6-1cd46d2ee00d"
+    },
+    "SemaphorePolicy": {
+      "deploy": false,
+      "semaphoreContract": "0x0000000000000000000000000000000000000000",
+      "groupId": 0
+    },
+    "MACI": {
+      "stateTreeDepth": 10,
+      "policy": "GitcoinPassportPolicy"
```

Comment:
Why `GitcoinPassportPolicy`? I can see other configs use `FreeForAllPolicy` here

I guess a related higher level question is - does all of the default config need to be the same for different chains? I can see a few other differences with other configs from other chains such in `EASPolicy`, `ZupassPolicy` etc